src/app/guards/ExpressWorkflowGuard/ConnectorGridSkeleton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Grid, Skeleton } from '@mui/material';
+
+import ConnectorSkeleton from 'src/components/connectors/Grid/Skeleton';
+
+export const ConnectorGridSkeleton = () => {
+    return (
+        <Grid
+            container
+            spacing={{ xs: 2, md: 3 }}
+            columns={{ xs: 4, sm: 4, md: 12, lg: 12, xl: 12 }}
```

Comment:
Why are `lg` and `xl` defined as `12`?

Reply:
The column specification does not cascade which was surprising given the handling of other MUI `Grid` props.